daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/preprocessing/Preprocessor.scala

```diff
@@ -54,7 +54,7 @@ private[engine] final class Preprocessor(compiledPackages: MutableCompiledPackag
                 )
               } yield r
             case None =>
-              ResultError(Error.Package.Generic(s"Couldn't find package $pkgId"))
+              ResultError(Error.Package.MissingPackage(pkgId))
```

Comment:
I’m a bit worried that there seem no tests for any of the other errors.  Am I missing something? At least self consistency seems worth testing somewhere™.

Reply:
I added tests and spotted/fixed a bug in 2ff96785562ac41f2793aff38ec889b6075595a0